uaclient/status.py

```diff
@@ -616,6 +622,20 @@ def __init__(
     docs_url=DOCUMENTATION_URL
 )
 
+NAMED_MESSAGE_FIPS_UPDATES_INVALIDATES_FIPS = NamedMessage(
+    "fips-updates-invalidates-fips",
+    "FIPS cannot be enabled if FIPS Updates has ever been enabled because"
+    " FIPS Updates installs security patches that aren't officially"
+    " certified.",
+)
+NAMED_MESSAGE_LIVEPATCH_INVALIDATES_FIPS = NamedMessage(
```

Comment:
That looks cool, but just to be sure, the idea would be to extend that to all messages to return the message code on the json response, right ?

Reply:
Potentially! I guess it would depend on what the desktop team needs

Reply:
I also think we should tackle internationalization with gettext next cycle, so alternatively maybe we could put this off until then